zulipterminal/model.py

```diff
@@ -331,13 +331,12 @@ def _start_presence_updates(self) -> None:
     def react_to_message(self,
                          message: Message,
                          reaction_to_toggle: str) -> None:
-        # FIXME Only support thumbs_up for now
```

Comment:
I think we can keep this FIXME comment till the commit we add support for all emoji reactions.